src/main/target/link/stm32_flash_split.ld

```diff
@@ -135,6 +144,14 @@ SECTIONS
     __fastram_bss_end__ = .;
   } >FASTRAM
 
+  .persistent_data :
```

Comment:
Maybe put `NOLOAD` here - clever debugger may reload zeroes, not sure if it is convenient or confusing ...